crates/bevy_sprite/src/render/sprite_sheet.vert

```diff
@@ -45,7 +45,7 @@ void main() {
         vec2(sprite_rect.end.x, sprite_rect.begin.y), 
         sprite_rect.end
     );
-    v_Uv = (atlas_positions[gl_VertexIndex] + vec2(0.01, 0.01)) / AtlasSize;
+    v_Uv = (atlas_positions[gl_VertexIndex]) / AtlasSize;
     v_Color = TextureAtlasSprite_color;
-    gl_Position = ViewProj * SpriteTransform * vec4(ceil(vertex_position), 1.0);
+    gl_Position = ViewProj * SpriteTransform * vec4(vertex_position, 1.0);
```

Comment:
These changes cause artifacts on 1x displays:

![image](https://user-images.githubusercontent.com/2694663/103431525-cd82a180-4b96-11eb-8ef6-5802142a25c3.png)

Reply:
I was playing around with the glyph atlas generation, and have something that seems to fix this.